framework/guid/model.py

```diff
@@ -20,6 +20,26 @@ class Guid(StoredObject):
     _id = fields.StringField(primary=True)
     referent = fields.AbstractForeignField()
 
+    @classmethod
+    def generate(self, referent=None):
+        while True:
+            # Create GUID
+            guid_id = ''.join(random.sample(ALPHABET, 5))
```

Comment:
Good point; I see now that this makes no changes to previous logic. This is fine as is.